terraform/fullnode/gcp/variables.tf

```diff
@@ -148,3 +148,28 @@ variable "enable_kube_state_metrics" {
   description = "Enable kube-state-metrics within monitoring helm chart"
   default     = false
 }
+
+variable "gke_enable_private_nodes" {
+  description = "Enable private nodes for GKE cluster"
+  default     = true
+}
+
+variable "gke_enable_autoscaling" {
+  description = "Enable autoscaling for the GKE cluster"
```

Comment:
probably better to rename this to `Enable node auto-provisioning` and add a link to it https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-provisioning.

Unfortunately enable autoscaling for the GKE cluster can mean many things :)